include/seqan3/argument_parser/detail/format_base.hpp

```diff
@@ -54,25 +56,27 @@ class format_base
                                  float,
                                  bool,
                                  char,
-                                 std::string>;
-        std::vector<std::string> names{"INT (8 bit)",
-                                       "UNSIGNED (8 bit)",
-                                       "INT (16 bit)",
-                                       "UNSIGNED (16 bit)",
-                                       "INT (32 bit)",
-                                       "UNSIGNED (32 bit)",
-                                       "INT (64 bit)",
-                                       "UNSIGNED (64 bit)",
-                                       "DOUBLE",
-                                       "FLOAT",
-                                       "BOOL",
-                                       "CHAR",
-                                       "STRING"};
+                                 std::string,
+                                 filesystem::path>;
+        std::vector<std::string> names{"signed 8 bit integer",
+                                       "unsigned 8 bit integer",
+                                       "signed 16 bit integer",
+                                       "unsigned 16 bit integer",
+                                       "signed 32 bit integer",
+                                       "unsigned 32 bit integer",
+                                       "signed 64 bit integer",
+                                       "unsigned 64 bit integer",
+                                       "double",
+                                       "float",
+                                       "bool",
+                                       "char",
+                                       "std::string",
+                                       "filesystem::path"};
```

Comment:
why is this not `std::filesystem::path`?

Reply:
because I in gcc7 `<filesystem>` is not available yet so I include `<seqan3/io/filesystem.hpp>` instead which is not in the std namespace..?

Reply:
It is only inconsistent with the std::string. And it seems like some user output which might lead to confusions about whether filesystem is a seqan3 thing or not. 
So filesystem is only our namespace alias for std::filesystem/std::experimental::filesystem. I am wondering if we should define the  experimental alias in std namespace if it is not available otherwise. But I am not sure if the standard permits this and then always refer to std::filesystem instead.

Reply:
I agree that we should alias std::filesystem. 
The following works with GCC7 and 8 for our filesystem.hpp:

```cpp
#pragma once

#if __has_include(<filesystem>)
#include <filesystem>
#else
#include <experimental/filesystem>
#endif // __has_include(experimental/filesystem)

#include <seqan3/core/platform.hpp>

#if !__has_include(<filesystem>)
namespace std
{
namespace filesystem = std::experimental::filesystem;
} // namespace std
#endif // __has_include(experimental/filesystem)

```
But we should probably put this into `/std/` then instead.
What do you think?

Reply:
Makes sense.

Reply:
I will change this if @h-2 agrees